src/content/docs/docs/integrations/google-genai.mdx

```diff
@@ -57,69 +69,901 @@ const ai = genkit({
 
 const response = await ai.generate({
   model: googleAI.model('gemini-2.5-flash'),
-  prompt: 'Tell me something interesting about Google AI.',
+  prompt: 'Explain how neural networks learn in simple terms.',
 });
 
-console.log(response.text());
+console.log(response.text);
 ```
 
 ### Text Embedding
 
 ```typescript
 const embeddings = await ai.embed({
-  embedder: googleAI.embedder('text-embedding-004'),
-  content: 'Embed this text.',
+  embedder: googleAI.embedder('gemini-embedding-001'),
+  content: 'Machine learning models process data to make predictions.',
+});
+
+console.log(embeddings);
+```
+
+### Image Generation
+
+```typescript
+const response = await ai.generate({
+  model: googleAI.model('imagen-4.0-generate-001'),
+  prompt: 'A serene Japanese garden with cherry blossoms and a koi pond.',
+});
+
+const generatedImage = response.media;
+console.log('Image generated:', generatedImage);
+```
+
+## Language Models
+
+You can create models that call the Google Generative AI API. The models support tool calls and some have multi-modal capabilities.
```

Comment:
Maybe rephrase, the user is not **creating** models, just calling them. Creating a reference to them at most.